src/controllers/BaseTalkController.php

```diff
@@ -81,14 +81,14 @@ public function getUserMapper($db, $request)
     /**
      * Get a single talk
      *
-     * @param  PDO      $db
      * @param  Request  $request
+     * @param  PDO      $db
      * @param  integer  $talk_id
      * @param  boolean $verbose
      *
      * @throws Exception if the talk is not found
      *
-     * @return TalkModelCollection
+     * @return false|TalkModel
```

Comment:
@exussum12 Good catch! I looked for other examples and found one more. Fixed via bf3d283187